doc/examples/edges/plot_circular_elliptical_hough_transform.py

```diff
@@ -105,9 +107,11 @@
 edges = canny(image_gray, sigma=2.0, low_threshold=0.55, high_threshold=0.8)
 
 # Perform a Hough Transform
-# The accuracy corresponds to the bin size of a major axis.
-# The value is chosen in order to get a single high accumulator.
-# The threshold eliminates low accumulators
+# The accuracy corresponds to the bin size of the histogram for minor axis lengths.
+# A higher `accuracy` value will lead to more ellipses being found, at the
+# cost of a lower precision on the minor axis length estimation.
+# A higher `threshold` will lead to less ellipses being found, filtering out those
+# that have fewer edge points on their perimeter.
```

Comment:
Sorry, I don't understand: What are 'edge points' here? Naively, I would think points on the perimeter...

Reply:
"Edge points" are nonzeros from the output of the Canny detector. So "edge points" here can be from any objects, not necessarily the ellipses, whereas the "perimeter" refers only to the ellipses.

I thought it was relatively clear because the output of canny is stored in the `edges` variable:  
```python 
edges = canny(image_gray, sigma=2.0, low_threshold=0.55, high_threshold=0.8)
```

I can try to clarify, like:

```python
# A higher `threshold` will lead to less ellipses being found, filtering out those
# that have fewer edge points (as found by the Canny detector) on their perimeter.
```

But I'm a bit afraid it's making an already heavy explanation heavier. I'll keep thinking on it and try to come up with something, but suggestions welcome :)

Reply:
Thank you, @adfoucart! You are right, `edges` is computed right above... The variable name isn't `edge_points` though, so let's make it super clear. :smile: I would suggest:
```
# A higher `threshold` will lead to less ellipses being found, filtering out those
# with fewer edge points (as found above by the Canny detector) on their perimeter.
```